.cirrus.yml

```diff
@@ -743,6 +743,40 @@ upgrade_test_task:
     always: *logs_artifacts
 
 
+image_build_task:
+    name: "Build multi-arch $CTXDIR"
+    alias: image_build
+    skip: *tags
```

Comment:
Is this `skip` really needed given the `only-if` below?

Reply:
Good catch, no it's not needed.  Though I s'pose it won't hurt anything either.

Reply:
Since you just re-pushed, could you re-push again and remove the `skip`? It hurts in the sense that it will mislead future maintainers.